### ISSUE TITLE
Incorrect episode-num

### ISSUE THREAD

**Issue by cazaa** [ukdtom/epg-dk.bundle#28]
It seems that Yousee changed the structure for season and episode. For instance episodes of Simpsons on TV3 Max all end up in season 1 even though they're not. 

The reason is that season information now is part of the description, and as a result it ends up in sub-title - like this:

`<sub-title lang="da">Sæson: 27</sub-title>`

I've added a small piece of code locally in `__init__.py` that parses the season info from `sub-title` and adds it to `episode-num`. The code looks like this:

```
if subtitle and subtitle[:7] == 'Sæson: ':
    Episode = Program['series_info']
    Total = subtitle[7:]
else:
    Episode = Program['series_info']
    Total = 1
```

Basically what the code does is it checks if the first 7 chars in `subtitle` is `Sæson: ` and adds the numeric value after that string as `Total`.

As a result I now get correct season and episode info in the XML.

In order for this fix to work, the Python script needs to know which encoding to use. This is done by adding the following as the first line of the script:

`# -*- coding: utf-8 -*-`

**Comment by ukdtom** [comment]
Care to fork, and make a PR?

**Comment by cazaa** [comment]
I've just created a pull request: https://github.com/ukdtom/epg-dk.bundle/pull/29

**Comment by ukdtom** [comment]
Saw it, merged and commented...Thanks mate
